### ISSUE TITLE
Grid Inputs

### ISSUE THREAD

**Issue by EddieDL** [PWNAGERobotics/ScoutingPASS#83]
How do you convert the value from the grid inputs into something that you can use? For example Column 3 Top.

**Comment by fuddster** [comment]
In the Excel directory is a VBA file that allows you to parse the data in Excel.   The code is in QRReader.bas.   You can see an example of it working in the 2023_grid_and_cycle_time_example.xlsm Excel file.

**Comment by zaaj** [comment]
In the version (FRC Team 95) modified for our own use, we changed the encoded data from:

field1=value1;field2=value2...

to:

 value1<tab>value2<tab>...
 
Barcode scanners act like a keyboards, and spreadsheet programs advance one cell to the right when Tab is typed.  We made sure our scanner appended an <enter> after each scan.  We just place the cursor in the left-most cell of the first empty row in our table, and scan the QR code - the data goes right into the cells with no script needed, and the cursor ends up in the correct cell to scan the next QR code.
